src/mudlet.pro

```diff
@@ -366,13 +366,13 @@ macx {
 }
 
 # use ccache if available
-unix {
```

Comment:
Some of the code (some ninth-party stuff), within the edbee-lib editor is C not C++ so do we want/need to frob the `QMAKE_C` variable as well.

As a side note can you make the `win32` case conditional on `contains( DEFINES, INCLUDE_MAIN_BUILD_SYSTEM )` **not** being true so that it doesn't act in a MSYS2/Mingw-w64 setup like mine where both `/mingw32/bin/cache.exe` and `/mingw64/bin/cache.exe` exist and I have to make sure I check the right one for the right *bitness*.

Reply:
Good idea

Reply:
I don't think your system will be affected because it is `/C/Program Files/ccache` that is checked - not `/mingw...`. So I will leave that check out for now.

Will add `QMAKE_C`.